white-paper.md

```diff
@@ -90,35 +90,35 @@ tea’s mission is to empower open-source communities and ensure their contribut
 
-Provided that the package maintainer offers such service, a package supporter and sponsor may receive a premium support level NFT in return, thus benefiting from accelerated SLAs or more flexible licensing. Additionally, package supporters and sponsors may decide to support packages or package maintainers and automatically redirect all or a percentage of their rewards to incentivize teams to build new open-source software. In other words, packages don’t need to exist for tea to start pouring in. Nascent projects can be supported just as well as more mature ones, further incentivizing a constantly evolving open-source landscape.
+Provided that the package maintainer offers such a service, a package supporter and sponsor may receive a premium support-level NFT in return, thus benefiting from accelerated SLAs or more flexible licensing. Additionally, package supporters and sponsors may decide to support packages or package maintainers and automatically redirect all or a percentage of their rewards to incentivize teams to build new open-source software. In other words, packages don’t need to exist for tea to start pouring in. Nascent projects can be supported just as well as more mature ones, further incentivizing a constantly evolving open-source landscape.
 
 ## tea Tasters
 
 As new packages or new versions of existing packages are released, the validity of the work needs to be provably demonstrated. This information is critical for package users to decide whether or not to trust both the package and its maintainers. With the tea protocol, this function is provided by the tea tasters.
 
-tea tasters, typically, are experienced software developers willing to dedicate some of their time to check the claims associated with a package (functionality, security, semantic versioning[^14], license accuracy, etc.) and stake both their reputation and economic value to demonstrate the outcome of their research and analysis and support their reviews. tea tasters receive rewards for their diligence and efforts. At tea, we call  “steeping your tea” the action of locking tea tokens to support your reviews and receive rewards (or penalties) based on the consensus on the validity of your reviews.
+tea tasters, typically, are experienced software developers willing to dedicate some of their time to check the claims associated with a package (e.g., functionality, security, semantic versioning[^14], license accuracy, etc.) and stake both their reputation and economic value to demonstrate the outcome of their research and analysis and support their reviews. tea tasters receive rewards for their diligence and efforts. At tea, we call  “steeping your tea” the action of locking tea tokens to support your reviews and receive rewards (or penalties) based on the consensus on the validity of your reviews.
```

Comment:
Fine with the addition of "e.g."

tea tasters could/should be experienced since they are reviewing code produced by others. Experience would typically help with the quality of the review. For example, one would likely look at a javascript code review produced by someone who's never written a line of javascript differently from one produced by someone who's published code in javascript.
We included "typically" to communicate that they should/could, but don't have to be. The reputation system will need to account for that.